src/edu/stanford/dlss/was/WasapiDownloader.java

```diff
@@ -1,8 +1,49 @@
 package edu.stanford.dlss.was;
 
+import java.io.*;
+import java.util.Properties;
+
 public class WasapiDownloader
 {
-    public static void main(String[] args) {
-        System.out.println("I am WasapiDownloader. Hear me roar. Woo.");
+  public static final String SETTINGS_FILE_LOCATION = "config/settings.properties";
+  public Properties settings = null;
+
+  public WasapiDownloader() {
+    loadPropertiesFile();
+  }
+
+  private void loadPropertiesFile() {
+    if (settings == null)
+    {
+      InputStream input = null;
+      try {
+        input = new FileInputStream(SETTINGS_FILE_LOCATION);
+        settings = new Properties();
+        settings.load(input);
+      } catch (IOException e) {
```

Comment:
I'm not sure if we should catch these exceptions.  Catching them and failing, didn't seem too bad an option